tests/test_decompiler.py

```diff
@@ -1767,8 +1788,8 @@ def test_decompiling_dd_argmatch_to_argument_noeagerreturns(self, decompiler_opt
             for p in all_optimization_passes
             if p is not angr.analyses.decompiler.optimization_passes.EagerReturnsSimplifier
         ]
-
```

Comment:
Don't add or remove random lines that cause the diff to be more messy. Remove or add all these lines back.